src/components/Strategies/StrategyList.tsx

```diff
@@ -86,6 +66,7 @@ const Container = styled.div`
   }
 
   .soon {
```

Comment:
Rename to more descriptive `comingSoon` + add some transition so it looks nicer when it appears, e.g. `transition: opacity 0.3s ease`